src/Microsoft.ML.Data/Transforms/NormalizeColumnSng.cs

```diff
@@ -605,10 +605,10 @@ public ImplVec(IHost host, TFloat[] scale, TFloat[] offset, int[] indicesNonZero
                     {
                     }
 
-                    public static new ImplVec Create(ModelLoadContext ctx, IHost host, ColumnType typeSrc)
+                    public static ImplVec Create(ModelLoadContext ctx, IHost host, VectorType typeSrc)
```

Comment:
>public static ImplVec Create(ModelLoadContext ctx, IHost host, VectorType typeSrc) [](start = 20, length = 82)

`public delegate void SignatureLoadColumnFunction(ModelLoadContext ctx, IHost host, ColumnType typeSrc);`
Is it works because VectorType is based on ColumnType, or we just don't have tests which covers loading of normalizer?

Reply:
I guess it works because it calls create method in NormalizeColumn.cs in which we call these methods.

---
In reply to: [247765979](https://github.com/dotnet/machinelearning/pull/2131#discussion_r247765979) [](ancestors = 247765979)

Reply:
I think @eerhardt 's code here represents an improvement, since it is more restrictive about when this was called, whereas the old code was, frankly, a bit too freewheeling about whether it was handling vector or scalar types... 😛